File: Python/Lista_01(Estrutura_Sequencial)/Troco.py
precoUnitarioProduto = float(input("Preco unitario do produto: "))

quantidadeComprada = int(input("Quantidade comprada: "))

dinheiroRecebido = float(input("Dinheiro recebido: "))

troco = dinheiroRecebido - (precoUnitarioProduto*quantidadeComprada)

print(f"TROCO = {troco:.2f}")

File: C++/Lista_01(Estrutura_Sequencial)/Troco.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){

    double precoUnitarioDoProduto, dinheiroRecebido, troco;
    int quantidadeComprada;

    cout << "Preco unitario do produto: ";
    cin >> precoUnitarioDoProduto;

    cout << "Quantidade comprada: ";
    cin >> quantidadeComprada;

    cout << "Dinheiro recebido: ";
    cin >> dinheiroRecebido;

    troco = dinheiroRecebido - precoUnitarioDoProduto*quantidadeComprada;

    cout << "TROCO = " << fixed << setprecision(2) << troco;

    return 0;

}